FLIR/conservator/generated/schema.py

```diff
@@ -4667,13 +4771,16 @@ class Mutation(sgqlc.types.Type):
                 (
                     "dataset_id",
                     sgqlc.types.Arg(
-                        sgqlc.types.non_null(ID), graphql_name="datasetId", default=None
+                        sgqlc.types.non_null(GraphqlID),
+                        graphql_name="datasetId",
+                        default=None,
                     ),
                 ),
                 (
                     "search_text",
                     sgqlc.types.Arg(String, graphql_name="searchText", default=None),
                 ),
+                ("sort", sgqlc.types.Arg(String, graphql_name="sort", default=None)),
```

Comment:
Out of curiosity, what does the "sort" field do for `datasetFrameIdFromIndex` ?

Reply:
A dataset frame does not have its own natural index, hence the need for `datasetFrameIdFromIndex`. Formerly, dataset frames were sorted by `videoId` and `frameIndex` (which is the index the frame had in its "parent video"); but we introduced sorting of dataset frames in the UI, so for the dataset annotator, we have to be able to figure out what the "next" frame is based on an index. Hence, this API needed to be able to take a sort parameter.